docs/reference/migration/migrate_6_2.asciidoc

```diff
@@ -23,3 +23,31 @@ existing <<jvm-options,`jvm.options`>>, you should add the line
 `-Djava.io.tmpdir=${ES_TMPDIR}`. It is safe to do this on all OS as we preserve
 using a private temporary directory on non-Linux systems with the same
 mechanism.
+
+=== GC logging
+
+Starting in Elasticsearch 6.2.0, JVM <<gc-logging,GC logging>> is enabled out of
+the box. This lightweight facility gives insight into GC issues. The settings
+for this are in <<jvm-options,`jvm.options`>> and default to keeping 2 GB worth
+(rotating every 64 MB) of logs in the default logs directory. If you are
+upgrading from a previous version of Elasticsearch and preserve your existing
+<<jvm-options,`jvm.options`>>, you should add the following GC logging settings
+so that this valuable debugging information is available:
+
+[source,sh]
+--------------------------------------------------
+-XX:+PrintGCDetails
+-XX:+PrintGCDateStamps
+-XX:+PrintTenuringDistribution
+-XX:+PrintGCApplicationStoppedTime
+-Xloggc:logs/gc.log or /var/log/elasticsearch/gc.log <1>
```

Comment:
I would put the "or" part in the description below, so that this can more easily be copy/pasted?

Reply:
I would rather copy-pasting not be encouraged here because the right value of this setting depends on whether or not a user is using a package distribution versus an archive distribution?